docs/example-scenario/index.md

```diff
@@ -55,7 +57,7 @@ Jump to: [AI scenarios](#ai-scenarios) | [Application scenarios](#application-sc
                     <div class="cardText">
                         <a href="https://azure-architecture.uservoice.com/forums/918625-architecture-guidance" data-linktype="external">
                             <div class="cardSize cardsF">
-                                <div class="cardPadding">
+                                <!-- <div class="cardPadding"> -->
```

Comment:
Is this div not needed?

Reply:
Must've hit the comment shortcut key by mistake. Reverted.